shared/compiler.go

```diff
@@ -271,6 +271,7 @@ func compileTimeLinkFiles(compilerExecName string, pr ParserResult, objFiles []s
 func buildObjectFile(compilerExecName string, pr ParserResult, srcFile string, objFile string) (success bool) {
 	args := pr.CompileArgs[:]
 	args = append(args, srcFile, "-c", "-o", objFile)
+	LogDebug("buildObjectFile: %v", args)
```

Comment:
N.B.: I left this in because I found it helpful, and thought it might help someone else in the future. But I'm happy to remove it 🙂